include/profiler.h

```diff
@@ -158,6 +158,22 @@ static inline void profiler_log_encode_u32(struct log_event_buf *buf,
 					   uint32_t data) {}
 #endif
 
+/** @brief Encode and add string to a buffer.
+	   Maximum 255 characters can be sent (the rest is ommited).
```

Comment:
Maybe we could move the second sequence outside `brief`. The `brief` should be a short one-liner.
We could even make it e.g. `warn`